package.json

```diff
@@ -1,32 +1,13 @@
 {
   "name": "esphome-webserver",
-  "version": "2.0.5",
-  "main": "index.js",
+  "version": "3.0.0",
```

Comment:
Though process is this is a relatively significant change to the structure of this repository, though this bump could wait for the upcoming v3 work instead. No requirement to bump it before or after.